### ISSUE TITLE
Test date picker on filter for search

### ISSUE THREAD

**Issue by cgsmith** [cgsmith/yii2-flatpickr-widget#2]

**Comment by DikkiX** [comment]
This was how it was done with Dosamigos Datepicker. maybe it can give u an inside.

` [
                'attribute' => 'Datum',
                'label' => 'Datum',
                'format' => 'raw',
                'value' => 'Datum',
                'filter'=>DatePicker::widget([
                'model' => $searchModel,
                'attribute' => 'Datum',
                'clientOptions' => [
                'autoclose' => true,
                'format' => 'yyyy-mm-dd'
            ]
            ])
            ],`

**Comment by cgsmith** [comment]
Thanks for this! I'll try later today.

**Comment by cgsmith** [comment]
Put this in your Gridview:

```php
 [
                'attribute' => 'created_at',
                'label' => 'Created At',
                'format' => 'datetime',
                'value' => 'created_at',
                'filter' => \cgsmith\flatpickr\FlatpickrWidget::widget(
                    [
                        'model' => $searchModel,
                        'attribute' => 'created_at',
                        'flatpickrConfig' => [
                                'enableTime' => false,
                                'dateFormat' => 'U',
                                'altInput' => true,
                                'altFormat' => 'F j, Y',
                        ],
                    ]
                )
            ],
            
```

**Comment by cgsmith** [comment]
Fixed in `1.0.1`
